bool MAPPING_RELEASE 
---
bool success
